src/components/cc-tile-requests/cc-tile-requests.types.d.ts

```diff
@@ -1,5 +1,25 @@
-type RequestsData = [
+export type RequestsData = [
   number, // Start timestamp in milliseconds. Expected to be rounded to the hour of its respective TZ.
   number, // End timestamp in milliseconds. Expected to be rounded to the hour of its respective TZ.
   number, // Number of request during this time window.
 ]
+
+export type TileRequestsState = TileRequestsStateLoaded | TileRequestsStateLoading | TileRequestsStateError | TileRequestsStateEmpty;
+
+export interface TileRequestsStateLoaded {
+  type: 'loaded';
+  data: Array<RequestsData>;
+}
+
+export interface TileRequestsStateLoading  {
+  type: 'loading';
+}
+
+export interface TileRequestsStateError {
+  type: 'error';
+}
+
+export interface TileRequestsStateEmpty {
```

Comment:
remark: initially, the `_empty` property was a private property. Emptiness should remain private so, I think that this 'empty' state should not exist. Instead, the emptiness should be (like initially) calculated from the `data` given with the 'loaded' state.

Reply:
see the other comment, I agree but I want @Galimede's opinion about it and I'd like to apply the same logic to the `cc-tile-metrics` if possible.